app/src/main/res/layout/fragment_department_c_t.xml

```diff
@@ -5,10 +5,401 @@
+                            android:textAlignment="center"
+                            android:textColor="#000068"
+                            android:textSize="16sp" />
+
+                        <TextView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginStart="15dp"
+                            android:layout_marginTop="5dp"
+                            android:layout_marginEnd="15dp"
+                            android:fontFamily="@font/roboto_light"
+                            android:text="Asst. Professor"
+                            android:textAlignment="center"
+                            android:textColor="#000062"
+                            android:textSize="16sp" />
+
+                    </LinearLayout>
+
+                    <LinearLayout
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="10dp"
+                        android:orientation="vertical">
+
+                        <ImageView
+                            android:layout_width="99dp"
+                            android:layout_height="97dp"
+                            android:layout_marginStart="50dp"
+                            android:layout_marginTop="15dp"
+                            android:src="@drawable/faculty_round_image_test" />
+
+                        <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center"
+                            android:layout_marginStart="15dp"
+                            android:layout_marginEnd="15dp"
+                            android:fontFamily="@font/roboto_bold"
+                            android:gravity="center"
+                            android:text="Dr. Kalpana Saha(Roy)"
+                            android:textAlignment="center"
+                            android:textColor="#000068"
+                            android:textSize="16sp" />
+
+                        <TextView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginStart="15dp"
+                            android:layout_marginTop="5dp"
+                            android:layout_marginEnd="15dp"
+                            android:fontFamily="@font/roboto_light"
+                            android:text="Asst. Professor"
+                            android:textAlignment="center"
+                            android:textColor="#000062"
+                            android:textSize="16sp" />
+
+                    </LinearLayout>
+
+                </LinearLayout>
```

Comment:
For this section please use LinearLayout weight property. You have given  all the views static width and height. This may affect the Ui for diff screen sizes.

Reply:
@cdhiraj40 would you please help @MohitGupta121 regarding that.

Reply:
> For this section please use LinearLayout weight property. You have given all the views static width and height. This may affect the Ui for diff screen sizes.

Okay I'll used weight property